lib/util.js

```diff
@@ -25,7 +25,8 @@ module.exports = /** @lends Util.prototype */ {
 
       while (parts.length > 0) {
         curr = path.resolve(curr, parts.shift());
-        if (! path.existsSync(curr)) {
+        var exists = fs.existsSync || path.existsSync;
```

Comment:
While in this instance it is unlikely things would go wrong, we should attempt to follow NodeJS best practices.  In this case, it would be just attempting to create the directory and handling the "already exists" error.  See http://nodejs.org/api/fs.html#fs_fs_exists_path_callback, 3rd paragraph.  This may even perform better than the two separate calls.  As of node v0.12.0, there is no path.exists function anymore, and the version in fs is also deprecated.